examples/angular/src/app/pages/pages.component.html

```diff
@@ -1,18 +1,11 @@
-@if (context()?.pageAsset; as pageAsset) {
 <div class="flex flex-col min-h-screen gap-6 bg-neutral-50">
-
-    @if(pageAsset.layout.header) {
```

Comment:
I remove this condition on example because pageAsset will not always have value 
In client side, inside UVE, this value will be an empty object, and the pageAsset data will come from UVE

Reply:
Can't be removed because the header can be disabled in the layout editor, so there are some cases where this property (and the footer) will come back as `false`